lib/process-services-cloud/src/lib/form/components/widgets/dropdown/dropdown-cloud.widget.scss

```diff
@@ -23,5 +23,14 @@
         &-dropdown-failed-message .adf-error-container {
             margin-top: 1px !important;
         }
+
+        .adf-select-filter-input {
```

Comment:
I'm not sure about this one, why we need this style applied now ? is it a new one?

Reply:
I cannot remember why I added this code. Probably there was some edge case but I cannot reproduce it now. I will remove this code.